pkg/sentry/fsimpl/kernfs/inode_impl_util.go

```diff
@@ -256,6 +256,13 @@ func (a *InodeAttrs) Stat(context.Context, *vfs.Filesystem, vfs.StatOptions) (li
 
 // SetStat implements Inode.SetStat.
 func (a *InodeAttrs) SetStat(ctx context.Context, fs *vfs.Filesystem, creds *auth.Credentials, opts vfs.SetStatOptions) error {
+	return a.SetInodeStat(ctx, fs, creds, opts)
```

Comment:
why create a new function if all it does is to do whatever the original one do

Reply:
Because the newly created `fuse.inode.SetStat()` function implements `kernfs.Inode.SetStat()` and shadows the original implementation of `kernfs.InodeAttrs.SetStat()` but I need it. So I move the implementation to another function and I can call it from `fuse` package to set attributes of `kernfs.InodeAttrs`.